openassessment/xblock/submission_mixin.py

```diff
@@ -783,11 +785,23 @@ def submission_path_and_context(self):
             context['save_status'] = self.save_status
             context['enable_delete_files'] = True
 
+            submit_enabled = True
+            if self.text_response == 'required' and not self.saved_response:
+                submit_enabled = False
+            if self.file_upload_response == 'required' and not file_urls:
+                submit_enabled = False
+            if self.text_response == 'optional' and self.file_upload_response == 'optional' \
+                    and not self.saved_response and not file_urls:
+                submit_enabled = False
+            context['submit_enabled'] = submit_enabled
+
             if self.teams_enabled:
                 try:
                     team_info = self.get_team_info()
                     if team_info:
                         context.update(team_info)
+                        if self.does_team_have_submission(team_info['team_id']):
+                            no_workflow_path = 'openassessmentblock/response/oa_response_team_already_submitted.html'
```

Comment:
Is there a reason to save this to `no_workflow_path` and not just update `path`?

Reply:
um .... now that I'm done with this, not really? at one point it was but then I moved something and it seems like it no longer is. (I'll stick it into our cleanup ticket so we can get these tickets closed)